projects/netgrif-components-core/src/assets/i18n/de.json

```diff
@@ -455,6 +455,7 @@
         "snackBar": {
             "deleteCanceled": "Prozess löschen abgebrochen",
             "deleteSuccess": "Prozess erfolgreich gelöscht",
+            "deleteInProgress": "Der ausgewählte Prozess wird jetzt gelöscht",
```

Comment:
The other two texts have a `.` at the end. This one does not

Reply:
done